src/Tests/Microsoft.NET.ToolPack.Tests/GivenThatWeWantToPackAToolProject.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+using NuGet.Packaging;
+
+namespace Microsoft.NET.ToolPack.Tests
+{
+    public class GivenThatWeWantToPackAToolProject : SdkTest
+    {
+        private readonly string _nugetPackage;
+
+        public GivenThatWeWantToPackAToolProject(ITestOutputHelper log) : base(log)
+        {
+            TestAsset helloWorldAsset = _testAssetsManager
+                .CopyTestAsset("PortableTool", "PackPortableTool" + Path.GetRandomFileName()) //TODO remvoe that, no checkin 
+                .WithSource()
+                .Restore(Log);
+
+            var packCommand = new PackCommand(Log, helloWorldAsset.TestRoot);
+
+            packCommand.Execute();
+
+            _nugetPackage = packCommand.GetNuGetPackage();
+        }
+
+        [Fact]
+        public void It_packs_successfully()
+        {
+            using (var nupkgReader = new PackageArchiveReader(_nugetPackage))
+            {
+                nupkgReader
+                    .GetToolItems()
+                    .Should().NotBeEmpty();
+            }
+        }
+
+        [Fact]
+        public void It_finds_the_entry_point_dll_and_command_name_and_put_in_setting_file()
+        {
+            using (var nupkgReader = new PackageArchiveReader(_nugetPackage))
+            {
+                IEnumerable<NuGet.Frameworks.NuGetFramework> supportedFrameworks = nupkgReader.GetSupportedFrameworks();
+                supportedFrameworks.Should().NotBeEmpty();
+
+                var tmpfilePath = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
```

Comment:
Use `TestAssetManager.GetTestDestinationDirectoryPath` to get a working folder for the tests instead of using something in the temp path (this applies elsewhere in this PR too).

Reply:
Fixed, use testroot, which is from TestAssetManager.GetTestDestinationDirectoryPath